scripts/generate_stats.py

```diff
@@ -73,6 +73,10 @@ def traffic_stats(config, ranges):
 def ga_stats(config, ranges):
     stats = {}
 
+    # We currently sample google analytics sessions, so we need to
+    # multiply to determine the true visitor count. e.g. '50' for 50%.
+    SAMPLE_MULTIPLIER = 100 / int(g.googleanalytics_sample_rate)
```

Comment:
This was an intentional choice at the time because I didn't want to deal with fractional visitors later, but I realize now that's dumb because it could make for some really off base calculations. I'll do the int conversion lower, thanks.